test/integration/components/index.spec.ts

```diff
@@ -0,0 +1,24 @@
+import * as path from 'path';
+import { ToolpadHome } from '../../models/ToolpadHome';
+import { test } from '../../playwright/test';
+import { readJsonFile } from '../../utils/fs';
+
+test('components', async ({ page }) => {
+  const dom = await readJsonFile(path.resolve(__dirname, './componentsDom.json'));
+  const { id: pageId } = Object.values(dom.nodes).find(
+    (node: any) => node.name === 'components',
+  ) as any;
+
+  const homeModel = new ToolpadHome(page);
+  await homeModel.goto();
+  const app = await homeModel.createApplication({ dom });
+
+  await page.goto(`/app/${app.id}/preview/pages/${pageId}`);
```

Comment:
should we have one of those Playwright Object Model classes for the runtime as we have for the home page and the editor? or is it maybe not worth it for the runtime?